spring-apps/autosleep-core/src/main/java/org/cloudfoundry/autosleep/ui/web/model/OrgEnrollmentConfigRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.cloudfoundry.autosleep.ui.web.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class OrgEnrollmentConfigRequest {
+
+    @JsonProperty("idle-duration")
+    private String idleDuration;
+
+    @JsonProperty("exclude-spaces-from-auto-enrollment")
+    private String excludeSpacesFromAutoEnrollment;
+
+    @JsonProperty("state")
+    private String state;
```

Comment:
Not necessarily. As discussed here https://www.pivotaltracker.com/n/projects/1891357/stories/134541301/comments/155991391,

**backoffice_enrolled** would mean `all spaces and apps inherit the values unless overridden`,
**backoffice_opted_out** `would mean all spaces and apps in that org EVEN IF enrolled separately (for example: you may have an autosleep service instance in a space of that org) would not be auto-slept.`
**Deleting** from the org enrollment table would mean the `space-level instances continue to work as is` (for example: if you have a service instance in a space of that org, that continues to work).

In other words, the backoffice_opted_out is an easy way of switching off autosleep for an entire org (even if there are service instances in its spaces) without unbinding the apps and deleting the service instances one-by-one. Enabling it for the whole org again is just another PUT request with backoffice_enrolled value.

Do you still think we don't need the above?